### ISSUE TITLE
rv64si/scall does not check epc.

### ISSUE THREAD

**Issue by ccelio** [riscv/riscv-tests#105]
I'm finding the logic a little hard to follow, but it looks like the `scall` test only checks `scause` but not `sepc`.

**Comment by ccelio** [comment]
Also, doesn't `TESTNUM, 1` mean `pass` (https://github.com/riscv/riscv-tests/blob/master/isa/rv64si/scall.S#L53)? Or am I misremembering that.

**Comment by aswaterman** [comment]
do you mind improving the test to check sepc?  I think it's just a matter of sticking a label on the scall instruction, then doing a ```la``` of that label in the exception handler and comparing it to sepc.
